src/tokens-studio/spectrum2-non-colors/spectrum2/typography.component/desktop.json

```diff
@@ -339,6 +339,16 @@
       }
     }
   },
+  "body-size-xxs": {
+    "value": "{font-size-50}",
+    "type": "fontSizes",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
```

Comment:
Just double-checking, that this token `body-size-xxs` is new for S2 and not existing in S1. Right?